docs/source/reproducers/03-layout.md

```diff
@@ -0,0 +1,155 @@
+# Zuul job
+## Words of warning
+This feature is still fresh, and limited. For now, you can reproduce only
+a job that doesn't rely on the content-provider job.
+
+It also has to run against the latest PR content output, since you won't
+have access to the actual commit hash on github.
+
+For now, it has been tested only against
+*podified-multinode-edpm-e2e-nobuild-tagged-crc* job.
```

Comment:
i think this bit won't age well (i.e. we'll soon start testing all the things.) wondering if you want to be specific like that

Reply:
for now, yes. Because I don't want ppl to lose time trying to reproduce a job that isn't supported ;)